lib/modules/platform/azure/index.spec.ts

```diff
@@ -1530,6 +1530,33 @@ describe('modules/platform/azure/index', () => {
       );
       await azure.addReviewers(123, ['test@bonjour.fr', 'jyc', 'required:def']);
       expect(azureApi.gitApi).toHaveBeenCalledTimes(3);
+      expect(logger.warn).toHaveBeenCalledTimes(1);
```

Comment:
did you forget to update tests?

Reply:
Fixed to check for the logger.once.info instead. Thank you